### PR TITLE
Add vue config for template updating

### PR DESCRIPTION
**Pull request by egaluza**

### CODE REVIEW

File: templates-generator/vue-config.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                    after: '[<%=^empty%>$1<%=/empty%>];'
+                }
+            ]
+        },
+        {
+            glob: 'src/router.js',
+            definitions: [
+                {
+                    before: /(import Home .*?)(import defaultLayout[^;]*)/s,
+                    after: '<%=^empty%>$1<%=/empty%>$2'
```
Comment (nightskylark) [inline templates-generator/vue-config.js:63]:
```suggestion
                    before: /((import (Home|Profile|DisplayData).*\n)+)/,
                    after: '<%=^empty%>$1<%=/empty%>'
```

File: templates-generator/vue-config.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                },
+                {
+                    before: /side-nav-(inner|outer)-toolbar/,
+                    after: '<%=layout%>'
+                },
+                {
+                    before: /(,)(\s+{\s+path: "\/".*redirect: "\/home"[^}]*})/s,
```
Comment (nightskylark) [inline templates-generator/vue-config.js:70]: Do we really need `(,)`here?

File: templates-generator/vue-config.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                    after: '$1<%=^empty%>$2<%=/empty%>'
+                },
+                {
```
Comment (nightskylark) [inline templates-generator/vue-config.js:73]: Why did you move this block to the top? It looks like it breaks the sequence logic.

Comment (nightskylark) [reply templates-generator/vue-config.js:73]: Also, it could leave a redundant comma at the end.

Comment (egaluza) [reply templates-generator/vue-config.js]: I replaced and changed this expression

File: templates-generator/vue-config.js (hunk 2 above)
Comment (egaluza) [reply templates-generator/vue-config.js]: Deleted (,)

File: templates-generator/vue-config.js (hunk 1 above)
Comment (egaluza) [reply templates-generator/vue-config.js]: Fixed

**Comment by egaluza** [comment]
[Add vue config for update-templates script](https://trello.com/c/paILLZAV/2173-add-vue-config-for-update-templates-script)
